manifests/frontend/load_balancer/website2.pp

```diff
@@ -54,6 +54,8 @@
     'frontend_fqdn' => $::fqdn,
   })
 
+  $local_config = hiera_hash("sunet_frontend_local", undef)
```

Comment:
Wouldn't it be better to have {} as the default so you don't try to merge undef into the config?

Reply:
Should be fixed in https://github.com/SUNET/puppet-sunet/pull/12/commits/e729ac21680d98ba6b2d888012959223e5b14467